webpack/assets/javascripts/react_app/components/common/table/formatters/translatedCellFormatter.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import cellFormatter from './cellFormatter';
+
+export default value => cellFormatter(__(value));
```

Comment:
strings are not really captured for translation on runtime,
you need to mark an actual string, not a variable in order to mark it for translation,
you can read more about the process in https://projects.theforeman.org/projects/foreman/wiki/Translating

Reply:
Does that matter in this case? Aren't the strings to be displayed loaded from the backend where they (hopefully) should be marked? So in this case it indeed couldn't be used to extract strings, but since they are extracted elsewhere i wouldn't matter and they would just get translated?

Reply:
if they are translated already in the backend,  I believe we should just display them in the table without any extra translation/marking.

Reply:
The descriptions are in files such as https://github.com/theforeman/foreman/blob/develop/app/registries/foreman/settings/general.rb and you can see they are marked for translation (using `N_()`, see https://projects.theforeman.org/projects/foreman/wiki/Translating#Extracting-strings):
https://github.com/theforeman/foreman/blob/f7e5525dc8a3af7dd5333038f633f4e8b462f5be/app/registries/foreman/settings/general.rb#L8

That means they are present in the extracted strings, but you need to choose when to translate them. I don't even think the API can translate them. Perhaps that would be a good thing, but will be a far larger effort and we already have all the translations present in the frontend code.

In short: the original `value => cellFormatter(__(value))` looks correct to me, provided there is some asterisk that the caller must ensure the input is extracted somewhere. I don't know what a good way is, since I don't know how we're supposed to document the frontend code, but the least is a code comment I guess.